modules/image-selector/README.md

```diff
@@ -0,0 +1,67 @@
+# Terraform IBM common utilities Image Selector
+
+This Terraform module retrieves the most recent IBM Cloud VPC image based on a specified operating system and architecture.
+
+This module enables filtering of IBM Cloud images based on operating system and system architecture. It currently supports `ubuntu` as the operating system and `amd64` or `s390x` as valid architecture options.
+
+By applying semantic sorting to image names, the module identifies and returns the most recent image available. The resulting image ID and name can be seamlessly integrated into downstream resources such as virtual server instance provisioning.
+
+> Note: IBM Cloud image IDs are region-specific. This module includes a provider block to ensure the correct region is set when querying image metadata.
```

Comment:
modules should never have provider blocks include in them. You just need to update the usage to include a provider block

Reply:
Done.